pkg/cli/mirror/operator.go

```diff
@@ -578,3 +586,10 @@ func parseRelatedImages(root string) (map[string]struct{}, error) {
 	delete(relatedImages, "")
 	return relatedImages, nil
 }
+
+func validate(dc *declcfg.DeclarativeConfig) error {
+	if dc == nil {
+		return errors.New("nil declarative config")
```

Comment:
Since dc should never be nil, I'd treat this as a bug so people know to file an issue
```suggestion
		return errors.New("bug: nil declarative config")
```